code/modules/mapping/random_rooms.dm

```diff
@@ -1478,3 +1478,41 @@
 	centerspawner = FALSE
 	template_height = 11
 	template_width = 5
+
+
+/datum/map_template/random_room/engines/box/supermatter
```

Comment:
And these
monkestation/code/modules/mapping/random_rooms.dm

Reply:
Unable to get this specific change to work. It does not want to create the spawner object when in that location.